openwell/questioner
# #163653735 user can vote on a question

#### What does this PR do?
- This PR integrates user voting on questions
#### Description of Task to be completed?
- When a user is registered and logged in. the user can navigate to meetups and view all meetups
- A user clicking on a meetup will show more details and display all questions available so a user can vote.
#### How should this be manually tested?
- Pull this branch [ft-user-able-vote-question-163653735](https://github.com/openwell/questioner/tree/ft-user-able-vote-question-163653735) off this repository
- Navigate to `meetups.html` and click on a meetup.
- then the details of the meetup should be displayed and you can then vote on a question of choice.
- a successful vote should increase the votes count. 
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
[#163653735](https://www.pivotaltracker.com/story/show/163653735)
#### Screenshots (if appropriate)
None
#### Questions:
None
